ReactApp/car_rental_react/src/components/EnterLocation/index.js

```diff
@@ -2,12 +2,13 @@ import React, { useEffect, useRef, useState } from "react";
 import "./EnterLocation.css";
 

```

Comment:
Please rename this file to its component (i.e: EnterLocation.js). Same with other file

Reply:
And maybe name it to something like LocationInput?